tests/func-tests/node_placement_test.go

```diff
@@ -36,28 +36,30 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
 		originalWorkloadSpec hcov1beta1.HyperConvergedConfig
 		cli                  client.Client
 		cliSet               *kubernetes.Clientset
+		workerNodes          *v1.NodeList
 	)
 
 	BeforeAll(func(ctx context.Context) {
 		cli = tests.GetControllerRuntimeClient()
 		cliSet = tests.GetK8sClientSet()
 
-		nodes := listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
-		tests.FailIfSingleNode(len(nodes.Items) < 2)
+		workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/worker")
+		//workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
+		tests.FailIfSingleNode(len(workerNodes.Items) < 2)
 
 		// Label all but first node with "node.kubernetes.io/hco-test-node-type=infra"
 		// We are doing this to remove dependency of this Describe block on a shell script that
 		// labels the nodes this way
 		Eventually(func(g Gomega, ctx context.Context) {
-			for _, node := range nodes.Items[:len(nodes.Items)-1] {
+			for _, node := range workerNodes.Items[:len(workerNodes.Items)-1] {
```

Comment:
nit: this is labeling all but **last** node in the list, as opposed to the comment above.